libr/core/linux_heap_glibc.c

```diff
@@ -1457,6 +1457,7 @@ static void GH(print_heap_mmaped)(RCore *core, GHT malloc_state, GHT global_max_
 }
 
 void GH(print_malloc_states)( RCore *core, GHT m_arena, MallocState *main_arena) {
+	const int tcache = r_config_get_i (core->config, "dbg.glibc.tcache");
```

Comment:
Is having a const-qualified variable initialized by a function standard C?

Reply:
i'll change it, thx.